src/config.js

```diff
@@ -276,9 +276,27 @@ export default class Config {
       networkConcurrency: this.networkConcurrency,
       networkTimeout: this.networkTimeout,
     });
-    this._cacheRootFolder = String(
-      opts.cacheFolder || this.getOption('cache-folder', true) || constants.MODULE_CACHE_DIRECTORY,
-    );
+
+    let cacheRootFolder = opts.cacheFolder || this.getOption('cache-folder', true);
+
+    for (let t = 0; t < constants.PREFERRED_MODULE_CACHE_DIRECTORIES.length && !cacheRootFolder; ++t) {
```

Comment:
Good question, but I don't really know 😆 I've been using it for single-level loop since I started programming (and `u`/`v`/`w` or `x`/`y`/`z` when multiple loops are combined).

Could be amusing to run a search on the github repos to find what variable name is preferred by developpers :)